PowerPointLabs/PowerPointLabs/TextCollection/ColorsLabText.cs

```diff
@@ -14,17 +14,16 @@ internal static class ColorsLabText
         public const string TaskPanelTitle = "Colors Lab";
 
         public const string MainColorBoxTooltip = "Choose the main color: " +
-                                   "\r\nDrag the button to pick a color, " +
+                                   "\r\nYou can drag the box to favorites palatte, " +
```

Comment:
Maybe start the sentence with "Drag the box..." so that it is more consistent?

Reply:
Done